docs/plugins/approved-plugins.rst

```diff
@@ -0,0 +1,30 @@
+Plugins Approved by Sylius
+==========================
+
+As the Sylius eCommerce framework is an open source project it has an awesome community of users and developers.
+Therefore our ecosystem flourishes with plugins created outside of our organization. These plugins can become officially
+approved by us, when they meet certain requirements. Then, when accepted, they will land on the `official list of plugins
+on our website <https://sylius.com/plugins/>`_.
+
+When a plugin is approved by Sylius, you can recognize it also by this badge below in its readme file:
+
+.. image:: ../_images/approved_plugin.png
+    :scale: 50%
+
+|
+
+How to have a Plugin approved by Sylius?
+----------------------------------------
+
+Since Sylius is an open-source platform, there is a certain flow in order for the plugin to become officially adopted by the community.
+
+**1.** Develop the plugin using :doc:`the official Plugin Development guide </plugins/creating-plugin>`.
+
+**2.** Remember about the tests and code quality!
+
+**3.** Send it to the project maintainers. It can be via email to any member of the Sylius Core team, or `the official Sylius Slack <http://sylius.com/slack>`_.
+
+**4.** One of our Plugin Curators will contact you with the feedback regarding your plugin's code quality, test suite,
```

Comment:
I'm thinking if it makes sense to list our Plugin Curators, or link to Holaspirit directly: https://app.holaspirit.com/public/sylius/circle/YzgQ1c-sylius-ecommerce-framework/role/fDFHpy-plugin-curator WDYT?